service/src/main/java/bio/terra/cbas/config/BeanConfig.java

```diff
@@ -41,4 +43,13 @@ public ObjectMapper objectMapper() {
   public BearerToken bearerToken(HttpServletRequest request) {
     return new BearerTokenFactory().from(request);
   }
+
+  @Bean("cromwellWriteClient")
+  @RequestScope
+  public cromwell.client.ApiClient cromwellWriteClient(
```

Comment:
Do we need the `HttpServletRequest` here?

Reply:
@cjllanwarne did we decide what to do with the `HttpServletRequest` here?

Reply:
I believe it's not needed here because it's already provided to the BearerToken bean constructor which has the same `RequestScope` annotation